tensorflow_lite_support/ios/task/audio/core/audio_tensor/sources/TFLAudioTensor.h

```diff
@@ -0,0 +1,95 @@
+ * appearing at the end of `buffer.data`.
+ */
+@property(nonatomic, readonly) TFLFloatBuffer *buffer;
+
+/** Capacit of the `TFLAudioTensor` buffer in number of elements. */
+@property(nonatomic, readonly) NSUInteger bufferSize;
+
+/**
+ * Initializes a new `TFLAudioTensor` with a given `TFLAudioFormat` and sample count.
+ *
+ * @discussion The `TFLAudioTensor` stores data in a ring buffer of size sampleCount *
+ * TFLAudioFormat.channelCount.
+ *
+ * @param format An audio format of type `TFLAudioFormat`.
+ * @param sampleCount The number of samples `TFLAudioTensor` can store at any given
+ * time. The sampleCount provided will be used to calculate the buffer size of `TFLAudioTensor`
+ * bufferSize = format.channelCount * sampleCount.
+ *
+ * @return A new instance of TFLAudioTensor with the given audio format and sample count.
+ */
+- (instancetype)initWithAudioFormat:(TFLAudioFormat *)format sampleCount:(NSUInteger)sampleCount;
+
+/**
+ * Convenience method to load the elements currently in the internal buffer of `TFLAudioRecord` into
+ * `TFLAudioTensor`.
+ *
+ * @discussion You must ensure that the  audio formats of `TFLAudioRecord` and the current
+ * `TFLAudioTensor` match.
+ * New data from the input buffer is appended to the end of the buffer by shifting out
+ * any old data from the beginning of the buffer if need be to make space. If the size of the new
+ * data to be copied is more than the capacity of the buffer, only the most recent data
+ * of the `TFLAudioTensor`'s buffer size will be copied from the input buffer.
+ *
+ * @param audioRecord  An object of `TFLAudioRecord`.
+ * @param error An optional error parameter populated with the reason for failure, if the internal
+ * buffer of `TFLAudioRecord` could not be loaded into the `TFLAudioTensor`.
+ * @return A boolean indicating if the load operation succeded.
+ */
+- (BOOL)loadAudioRecord:(TFLAudioRecord *)audioRecord
+              withError:(NSError **)error NS_SWIFT_NAME(loadAudioRecord(audioRecord:));
+
+/**
+ * This function loads the internal buffer of `TFLAudioTensor` with the provided buffer.
+ *
+ * @discussion New data from the input buffer is appended to the end of the buffer by shifting out
+ * any old data from the beginning of the buffer if need be to make space. If the size of the new
+ * data to be copied is more than the capacity of the buffer, only the most recent data
+ * of the `TFLAudioTensor`'s buffer size will be copied from the input buffer.
+ *
+ * @param sourceBuffer  A buffer of type `TFLFloatBuffer`. For multi-channel
+ * input, the array must be interleaved.
+ * @param offset Starting index in the source buffer from which elements should be copied.
+ * @param size The number of elements to be copied.
+ * @param error An optional error parameter populated with the reason for failure, if the internal
+ * buffer could not be loaded with the source buffer.
+ *
+ * @return A boolean indicating if the load operation succeded.
```

Comment:
```suggestion
 * buffer could not be loaded with the source buffer.
 * @return A boolean indicating if the load operation succeded.
```

Reply:
There was a mismatch between the spacing of the documentation of various functions in this file. My bad.
I intended to follow the format of [TFLInterpreter.h](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/objc/apis/TFLInterpreter.h). According to this file the space should be retained. I have updated the documentation of all functions in [TFLAudioTensor](https://github.com/tensorflow/tflite-support/pull/832/files/b57efe97d6bfe8710b52a02d2a3a7d6fdaaa2b35#diff-795194cf478a2a4f6881ace619b030aabdeb9f5d8c0b4b07bfd503a5c00bacab) to match the aforementioned file.
LMK if any other format needs to be followed.